Directory.Build.props

```diff
@@ -1,35 +1,36 @@
 <Project>
-
   <PropertyGroup>
-    <_IsGitHub Condition="'$(GITHUB_ACTIONS)' == 'true'">true</_IsGitHub>
-    <_ContinuousIntegrationBuild Condition="'$(TF_BUILD)' == 'true' or '$(GITHUB_ACTIONS)' == 'true' or '$(GITLAB_CI)' == 'true'">true</_ContinuousIntegrationBuild>
-
     <!-- Assembly Info -->
     <Company>Ubisoft</Company>
     <Copyright>Copyright © $([System.DateTime]::UtcNow.Year) $(Company)</Copyright>
     <Description>GitLab REST API .NET Client Library</Description>
     <Product>NGitLab</Product>
+
     <!-- Compile Options -->
     <LangVersion>9.0</LangVersion>
     <Deterministic>true</Deterministic>
     <Features>strict</Features>
     <TreatWarningsAsErrors Condition="'$(Configuration)' != 'Debug'">true</TreatWarningsAsErrors>
 
     <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
+	
+	<!-- Package info -->
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
   </PropertyGroup>
+  
+  <ItemGroup>  
+    <None Include="$(MSBuildThisFileDirectory)\LICENSE" Pack="true" PackagePath="" Visible="false" />
+    <None Include="$(MSBuildThisFileDirectory)\README.md" Pack="true" PackagePath="" />
+	
+    <SourceRoot Include="$(MSBuildThisFileDirectory)/" />
 
-  <!-- SourceLink: https://www.meziantou.net/how-to-debug-nuget-packages-using-sourcelink.htm -->
-  <PropertyGroup Condition="'$(_IsGitHub)' == 'true'">
```

Comment:
How come this whole section is gone?

Reply:
The package `DotNet.ReproducibleBuilds` includes everything :)

You can check the props/target files in https://github.com/dotnet/reproducible-builds/tree/main/src/DotNet.ReproducibleBuilds